config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -0,0 +1,137 @@
+periodics:
+# Daily run, to enforce the state of the manifest on a regular basis. This could
+# happen more frequently (perhaps hourly), but it's probably not a big deal,
+# given how the postsubmit will take care of all needs (in theory). This is more
+# of a sanity check in cases where there are no merges to "master" branch for
+# long periods of time.
+- interval: 24h
+  max_concurrency: 1
+  # This name is the "job name", passed in as "--job=NAME" for mkpj.
+  name: cip-sanity
+  # Enable Pod Utilities. See https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md.
+  decorate: true
+  extra_refs:
+  # Because of Pod Utilities, we clone the below repo automatically, and, get
+  # dropped into /home/prow/go/src/github.com/kubernetes/k8s.io.
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: master
+  spec:
+    containers:
+    # TODO: Move the official cip image to a more serious location. But it's not
+    # a huge problem because we pin it by the digest.
+    - image: gcr.io/cip-demo-staging/cip@sha256:96be240bb072cb6b2b998e29daf87e1de4cd18c18d6ea97bec3f0cce487087ed
+      args:
```

Comment:
I'm guessing this relies on the default entry-point ? 
Otherwise you want something like `command: /bin/sh` and rest as args. 
Also, we can call `multirun.sh` directly? No need for `/bin/sh -c`

Similar everywhere else.